### PR TITLE
React-boilerplate

### PR DESCRIPTION
**Pull request by Brandon-MK**
Changes:
1. Adding the react boilerplate

### CODE REVIEW

**Comment by heisfullstacked** [comment]
Change the PR title. Add description mentioning what you did.

File: README.md [hunk 1]
```diff
@@ -1 +1 @@
-# Personal-diary-React
\ No newline at end of file
+# Personal-diary-React
```
Comment (heisfullstacked) [inline README.md:1]: Add some instructions about how to setup the repo, install the dependencies and run the webapp.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "personal-diary",
+  "version": "0.1.0",
+  "private": true,
```
Comment (heisfullstacked) [inline package.json:4]: false

**Review by heisfullstacked — CHANGES_REQUESTED** [review]

**Comment by Brandon-MK** [comment]
@heisfullstacked i have made the changes you told me to do!

**Review by heisfullstacked — APPROVED** [review]

**Pull request merged**